build/docker/server/Dockerfile.el9

```diff
@@ -24,7 +24,7 @@ WORKDIR /opt
 RUN microdnf -y install yum && yum -y install python3-pip && \
     yum -y install oracle-epel-release-el9 ansible-core && \
     yum -y install epel-release && \
-    yum -y install ansible
+    yum -y install ansible vi
```

Comment:
Let's install `vim` instead

Reply:
Vim is much heavier in terms of binary size. Given that we install it to the base image, I was trying not to blow up the image size.

Reply:
Agreed to go ahead with `vi`.